packages/grafana-ui/src/components/Logs/logParser.ts

```diff
@@ -108,10 +108,6 @@ function shouldRemoveField(field: Field, index: number, row: LogRowModel) {
   if (field.name === 'id') {
     return true;
   }
-  // entry field which we are showing as the log message
-  if (row.entryFieldIndex === index) {
```

Comment:
If I do this, then I should edit `grafana-ui/src/components/Logs/LogDetails.tsx` as well - is that what we want? This doesnt seem to be used by explore, so maybe I should not touch this file?

Reply:
@ifrost just want to confirm, since we aren't editing anything else in grafana-ui/logs, this should be added back?